run-trufflehog.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# Change /path/to/your/project to your actual project directory
```

Comment:
As noted above, probably just delete this script

Reply:
Removed thanks